applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -32,9 +32,16 @@ def __init__(self, Model, settings):
         self.epsilon = settings["epsilon"].GetDouble()
 
         self.fluid_model_part = self.body_model_part.GetRootModelPart()
-        self.trailing_edge_model_part = self.fluid_model_part.CreateSubModelPart("trailing_edge_model_part")
-        #List to store trailing edge elements id
+        if not self.fluid_model_part.HasSubModelPart("trailing_edge_model_part"):
+            self.trailing_edge_model_part = self.fluid_model_part.CreateSubModelPart("trailing_edge_model_part")
+        else: self.trailing_edge_model_part = self.fluid_model_part.GetSubModelPart("trailing_edge_model_part")
+
+        if not self.fluid_model_part.HasSubModelPart("wake__elements"):
```

Comment:
Here "wake__elements" has two underscores instead of 1
```suggestion
        if not self.fluid_model_part.HasSubModelPart("wake_elements"):
```